scripts/build_images.sh

```diff
@@ -19,4 +19,5 @@ docker build -t php-nginx php-nginx
 docker build -t php56 testapps/php56
 docker build -t php56_custom testapps/php56_custom
 docker build -t php70 testapps/php70
+docker build -t php56_70 testapps/php56_70
```

Comment:
This is a virtual user app, which has a composer entry of php "^5.6|^7.0". For now we should choose PHP 5.6 for such apps.